calicoctl/calicoctl/commands/version.go

```diff
@@ -87,6 +90,11 @@ Description:
 	cf := parsedArgs["--config"].(string)
 	client, err := clientmgr.NewClient(cf)
 	if err != nil {
+		if derr, ok := err.(errors.ErrorDatastoreError); ok {
```

Comment:
Looks like we will get this error if we hit an actual etcdv3 error as well: https://github.com/projectcalico/calico/blob/3ca528bed2d9af244d70ca7cb581d0af107e148b/libcalico-go/lib/backend/etcdv3/etcdv3.go#L381-L386

However, that's probably OK because the message we print here is appropriate for that scenario :)